src/app.ts

```diff
@@ -1,6 +1,72 @@
+type ResultData = {
+    id: string,
+    gender: string,
+    age: number,
+    row: number,
+}
+
+type PagingData = {
+    next?: string,
+    previous?: string,
+}
+
+let previousBtn: HTMLElement | any = document.querySelector('button[data-prevbtn]');
+let nextBtn: HTMLElement | any = document.querySelector('button[data-nextbtn]');
+let pageView: HTMLElement | any = document.querySelector('label[data-pageview]');
+let tableBody: HTMLElement | any = document.querySelector('tbody[data-sink]');
```

Comment:
Codacy has a fix for the issue: [Replace `'tbody[data-sink]'` with `"tbody[data-sink]"`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
let tableBody: HTMLElement | any = document.querySelector("tbody[data-sink]");
```